pkg/generators/rules/OWNERS

```diff
@@ -0,0 +1,4 @@
+reviewers:
+- roycaihw
+approvers:
+- roycaihw
```

Comment:
we should have more than one approver to make sure one person is not bottleneck. You want to go on vacations, right? :) I suggest at least at me and lavalamp here.

Reply:
my assumption is the ONWERS here overwrites the top level one, if that is not the case,  ignore my comment.

Reply:
it automatically inherits ownership from parent OWNERS. We're good!